tests/flow/test_graph_create.py

```diff
@@ -194,3 +196,30 @@ def test10_match_duplicated_reltype(self):
             result = redis_graph.query(query)
             expected_result = [[0]]
             self.env.assertEquals(result.result_set, expected_result)
+
+    # test creating properties using node created previously in the same query
+    def test11_create_property_from_node(self):
+        # test using function with valid arguments
+        queries = [
+            "CREATE (a:A), (b)-[:R {k:toJSON(a)}]->(c)",
+            "CREATE (a:A), (b)-[:R]->(c {k:toJSON(a)})",
+        ]
+        for query in queries:
+            result = redis_graph.query(query)
+            self.env.assertEquals(result.nodes_created, 3, depth=1)
+            self.env.assertEquals(result.properties_set, 1, depth=1)
+            self.env.assertEquals(result.relationships_created, 1, depth=1)
+
+        # test using functions with invalid arguments
```

Comment:
please explain why do these queries are invalid

Reply:
OK. Comments added to code in [68c6c25](https://github.com/RedisGraph/RedisGraph/pull/3080/commits/68c6c25d22d52619ec76bf40bca5537fad244895)